src/components/Modal/Translate.vue

```diff
@@ -20,26 +20,44 @@
   -->
 
 <template>
-	<NcModal @close="$emit('close')">
+	<NcModal size="large" @close="$emit('close')">
 		<div class="translate-dialog">
 			<h2>{{ t('text', 'Translate') }}</h2>
-			<textarea v-model="input" />
-			<div class="language-selector">
-				<label for="fromLanguage">{{ t('text', 'Source language') }}</label>
-				<NcSelect v-model="fromLanguage"
-					input-id="fromLanguage"
-					:options="fromLanguages"
-					:append-to-body="false" />
+			<em>{{ t('text', 'You can translate parts of the document by selecting the text before using the translate function.') }}</em>
+			<div class="wrapper">
+				<div class="col">
+					<div class="language-selector">
+						<label for="fromLanguage">{{ t('text', 'Translate from') }}</label>
+						<NcSelect v-model="fromLanguage"
+							input-id="fromLanguage"
+							:placeholder="t('text', 'Select language')"
+							:options="fromLanguages"
+							:disabled="disableFromLanguageSelect"
+							:append-to-body="false" />
+					</div>
+					<textarea ref="input"
+						v-model="input"
+						autofocus="true"
```

Comment:
`autofocus` doesn't support a value, see https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus

```suggestion
						autofocus
```